rosapi/ip.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+
+class IP():
```

Comment:
I think we need to do some renaming here.  When we use the IP class, there are lots of sub paths that checkip could be referring to... e.g. /ip/dhcp-client/address etc.  

I think we may even want to consider an entire subclass called: 
ip.address

Note ip is all lower case, just so that we are matching the naming conventions of upstream mikrotik naming.